app/controllers/manageBusinesses/add/AddBusinessNameController.scala

```diff
@@ -59,10 +59,10 @@ class AddBusinessNameController @Inject()(val authorisedFunctions: AuthorisedFun
 
   private def getPostAction(isAgent: Boolean, isChange: Boolean): Call = {
     (isAgent, isChange) match {
-      case (false, false) => routes.AddBusinessNameController.submit()
-      case (false, _) => routes.AddBusinessNameController.submitChange()
-      case (_, false) => routes.AddBusinessNameController.submitAgent()
-      case (_, _) => routes.AddBusinessNameController.submitChangeAgent()
+      case (false, false) => routes.AddBusinessNameController.submit(false, false)
```

Comment:
refactor this like was done with the others should work
`      routes.AddBusinessNameController.submit(isAgent, isChange)`